### DISCUSSION TITLE
LMS 8.5.1 / Squeezebox Classic fast forward Podcast

### DISCUSSION THREAD

**Discussion by LMSSonos** [LMS-Community/slimserver#1096] [Q&A] [answered]
Hello

I just was wondering if the fast forward within a podcast with the squeezebox classic is not enabled? I can forward via remote controller app like Squeeze Ctrl, but not on the classic remote. 

Am I missing something?

Thanks

**Comment by michaelherger** [comment, marked as answer]
Could you please ask this question on https://forums.slimdevices.com - you'll find a great, helpful community there.

**Reply by LMSSonos** [reply]
Thanks for your awesome work and  pointing me to the right resource

So according a bit older info https://forums.slimdevices.com/forum/user-forums/mysqueezebox-com/31514-bug-fast-foward-does-not-work-in-podcasts?p=319038#post319038 seems not to be possible to search in podcasts.

**Reply by michaelherger** [reply]
Oh... a lot has changed in the past 17 years 😂. You better ask in the forum. There are guys who know the streaming specifics much better than I do. If it's working in a smartphone app, then I'd expect it to work on any interface!

**Reply by LMSSonos** [reply]
Just for completeness: Found out, when using the original Squeezebox Classic remote, FFWD works as expected, when using the Boom remote, it will not work and just jump to the beginning of the podcast
